pkg/cloudprovider/providers/openstack/openstack.go

```diff
@@ -106,6 +108,7 @@ type Config struct {
 		ApiKey     string `gcfg:"api-key"`
 		TenantId   string `gcfg:"tenant-id"`
 		TenantName string `gcfg:"tenant-name"`
+		TrustId    string `gcfg:"trust-id"`
```

Comment:
you need to create a specific Kubernetes user, you will leak the specific user creds but we don't care since it was created for this on purpose, and you can revoke the trust delegation between the specific user and yourself whenever you want : in this case the specific user become powerless.
http://blogs.rdoproject.org/5858/role-delegation-in-keystone-trusts